.github/workflows/integration.yml

```diff
@@ -1,6 +1,9 @@
-name: Integration
+name: CI
```

Comment:
Calling this Continuous Integration is probably accurate - same concern as above though - I'd prefer not having an acronym.

Reply:
It was my attempt to reduce the length of the name so that everything shows up nicely like this
![image](https://user-images.githubusercontent.com/36203857/84685703-8e894900-aeef-11ea-99c2-c8c11f9eeaf0.png)
But I can change this back

Reply:
That does look nice, what about cutting back on the job names - "Build, Deploy Dev & Zap"?